client/src/components/lectureItem/lectureItemComponent.js

```diff
@@ -0,0 +1,11 @@
+const lectureItemController = require('./lectureItemController');
+
+var lectureItemComponent = {
+    bindings: {
+        lectures: '=?'
```

Comment:
got the main different, thanks
that was implemented as tested feature, and there was a lectureList instead
remade it to real lectureItem component